src/khoj/utils/initialization.py

```diff
@@ -303,4 +305,19 @@ def _update_chat_model_options():
                 logger.error(f"🚨 Failed to create chat configuration: {e}", exc_info=True)
     else:
         _update_chat_model_options()
-        logger.info("🗣️ Chat model configuration updated")
+        logger.info("🗣️ Chat model options updated")
+
+    # Update the default chat model if it doesn't match
+    chat_config = ConversationAdapters.get_default_chat_model()
+    env_default_chat_model = os.getenv("KHOJ_CHAT_MODEL")
```

Comment:
nit: specify default. Update `run_evals.yml` accordingly.

```suggestion
    env_default_chat_model = os.getenv("KHOJ_DEFAULT_CHAT_MODEL")
```